CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Master (Unreleased)
+* Add `RSpec/VerifiedDoubleReference` cop. ([@t3h2mas][])
 
 * Fix a false positive for `RSpec/EmptyExampleGroup` when expectations in case statement. ([@ydah][])
```

Comment:
```suggestion

* Add `RSpec/VerifiedDoubleReference` cop. ([@t3h2mas][])
* Fix a false positive for `RSpec/EmptyExampleGroup` when expectations in case statement. ([@ydah][])
```